contrib/src/main/java/com/datatorrent/contrib/enrich/FixedWidthFSLoader.java

```diff
@@ -0,0 +1,248 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.contrib.enrich;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import javax.validation.constraints.NotNull;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.hadoop.classification.InterfaceStability;
+
+import com.google.common.collect.Maps;
+import com.google.common.primitives.Ints;
+import com.univocity.parsers.fixed.FixedWidthFieldLengths;
+import com.univocity.parsers.fixed.FixedWidthParser;
+import com.univocity.parsers.fixed.FixedWidthParserSettings;
+
+import com.datatorrent.contrib.parser.AbstractCsvParser.FIELD_TYPE;
+import com.datatorrent.contrib.parser.AbstractCsvParser.Field;
+
+/**
+ * This implementation of {@link FSLoader} is used to load data from fixed width
+ * file.User needs to set {@link FixedWidthFSLoader#fieldDescription} to specify
+ * field information.
+ */
+@InterfaceStability.Evolving
+public class FixedWidthFSLoader extends FSLoader
+{
```

Comment:
Following 3 fields (if they're parameters to FixedWidthFSLoader) should not be transient:
1. hasHeader
2. fieldDescription
3. padding

If they're made transient, they'll not get serialized and hence when the operator gets deployed in real apex environment, the variables will remain null and fail.